builtin/rebase.c

```diff
@@ -174,12 +183,15 @@ int cmd_rebase(int argc, const char **argv, const char *prefix)
 	int ret, flags, quiet = 0;
 	struct strbuf msg = STRBUF_INIT;
 	struct strbuf revisions = STRBUF_INIT;
+	struct strbuf quiet_config = STRBUF_INIT;
 	const char *restrict_revision = NULL;
 	struct object_id merge_base;
 	struct option builtin_rebase_options[] = {
 		OPT_STRING(0, "onto", &options.onto_name,
 			N_("revision"),
 			N_("rebase onto given branch instead of upstream")),
+		OPT__QUIET(&options.quiet_option,
```

Comment:
The `OPT__QUIET()` construct requires a pointer to an `int` to store its result, not a `const char *`...